html/YANPIWS.js

```diff
@@ -29,20 +29,21 @@ function loadXMLDoc(URL, targetId, callback) {
 }
 
 /**
- * handle resizing clock to big then small
+ * handle resizing clock to big then small 
+ * by toggling between big_time and small_time classes
  */
-// todo - use class for defaultSize, remove from signature
 function setClockSize(state, defaultSize){
     if (state == 'big'){
-        // todo - put all this in a class and then just add and remove class
-        $('#time').css('font-size', '127pt').css('text-align', 'center').css('width', '100%');
-        $('#date').css('text-align', 'center').css('width', '100%');
-        $('.big_clock_hide').hide();
+        var small_times = $('.small_time')
+        small_times.addClass("big_time")
+        small_times.removeClass("small_time")
+
+        console.log('using big size');
```

Comment:
```suggestion
        $('.small_time').addClass("big_time")
        $('.small_time').removeClass("small_time")
```

Less code, less debug ;)

Reply:
I'm curious about general jquery performance guidelines. My habit as a programmer is to puts a list into a variable (in this case a very small list) so that the list doesn't have to be generated on every call to jQuery('.smal_time'). Either way, I don't mind being more succinct :)

Reply:
Hmm - I don't know! I'm guessing jQuery is pretty performant either way given it's already extracting IDs and classes into it's own `$('.class')` and   `$('#ids')` already? 

And I'm being pretty pedantic about this change, sorry :(  Especially given how sloppy I feel the rest of this code is.  (and now I realize we should have done `$('.small_time').addClass("big_time")..removeClass("small_time");` :shrug: )

But! We're missing `;` on all the end of lines here - mostly safe, but can be dangerous in some situations. Maybe make it a one liner w/ semicolons?!? Either way, please add semicolons ;)